tests/publiccloud/prepare_tools.pm

```diff
@@ -27,15 +27,15 @@ sub run {
 
     if (is_sle) {
         my $modver = get_required_var('VERSION') =~ s/-SP\d+//gr;
-        add_suseconnect_product('sle-module-public-cloud', $modver);
+        add_suseconnect_product('sle-module-public-cloud', $modver, get_required_var('ARCH'));
     }
 
-
-    my $tools_repo = get_var('PUBLIC_CLOUD_TOOLS_REPO', '');
-    if ($tools_repo eq '') {
-        $tools_repo = 'http://download.opensuse.org/repositories/Cloud:/Tools/' . generate_version() . '/Cloud:Tools.repo';
+    my $default_tools_repos = 'http://download.suse.de/ibs/Devel:/PubCloud:/CI/' . generate_version() . '/Devel:PubCloud:CI.repo';
+    $default_tools_repos .= ' https://download.opensuse.org/repositories/devel:/languages:/python:/backports/' . generate_version() . '/devel:languages:python:backports.repo';
```

Comment:
In this case `PUBLIC_CLOUD_TOOLS_REPO` is not holding "tools repo" anymore. also it would be hardly readable in openQA UI . if you really want to go with the loop here , use approach used in similar situation in openQA : one variable for each repo and variables having similar naming pattern ( 'PC_REPO_1' , 'PC_REPO_2'  ... 'PC_REPO_X' ) 
But I would say that only for two repos is too much  , I would go with just separate variable PUBLIC)CLOUD_PY_REPO and that's it . ( less code less bugs :) )

Reply:
The reason why  I went with such solution is, We only change the repo, but keep the `zypper in` call the same. So I was looking for a flexible solution, to change the repos again (which will happen for sure :/ ) without introduce more usless variables.

Reply:
I'm ok with this approach, even though it doesn't look perfect. Let's not have many variables for repos in the job level. If we are lucky and all the dependencies are resolved we won't need to use the backports repo and we will have only 1 element in the array and the loop still works...